src/interface/ui/accountactivity/BlockchainAccountActivity.cpp

```diff
@@ -308,7 +308,7 @@ auto BlockchainAccountActivity::load_thread_portion_p() noexcept -> bool
 
     PTimer<Diag>::start(Diag::portion_p);
     tdiag("PPP 1");
-    std::vector<std::future<unsigned>> fvec = [this]() {
+    std::vector<std::future<unsigned>> fvec = [=]() {
```

Comment:
Another way would be [this,threadcount,stride] - a bit more verbose but the compiler will avoid copying several of the data fields that the lambda needs.